server/lib/tuist/qa/agent.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Tuist.QA.Agent do
+  @moduledoc """
+  Tuist QA agent module.
+  """
+
+  alias Tuist.QA.SimulatorController
```

Comment:
I find the naming convention a bit Swift-inspired. What about `Tuist.QA.Simulators`?